### PR TITLE
Document the Jsonnet example

### PR DESCRIPTION
**Pull request by imobachgs**
Add some comments to the Jsonnet-based example.

### CODE REVIEW

File: rust/agama-lib/share/examples/profile.jsonnet [hunk 1]
```diff
@@ -1,4 +1,10 @@
+// This is a Jsonnet file. Please, check https://jsonnet.org/ for more
+// information about the language.
+
+// The hw.libsonnet file contains hardware information from the "lshw" tool.
```
Comment (mvidner) [inline rust/agama-lib/share/examples/profile.jsonnet:4]: On  Leap 15.4 at least, lshw.rpm or jsonnet.rpm do not contain a hw.libsonnet file. Where do I find it or how do I know what API it provides?

File: rust/agama-lib/share/examples/profile.jsonnet [hunk 2]
```diff
@@ -1,4 +1,10 @@
+// This is a Jsonnet file. Please, check https://jsonnet.org/ for more
+// information about the language.
```
Comment (mvidner) [inline rust/agama-lib/share/examples/profile.jsonnet:2]: OK, that's about the syntax.
As a user I'd also want to know the schema, so 

```suggestion
// information about the language.
// For the schema, see
// https://github.com/openSUSE/agama/blob/master/rust/agama-lib/share/profile.schema.json
```

**Review by mvidner — APPROVED** [review]
Thanks!

`lshw` makes sense now, it contains the *data* about the current system. Somehow I tricked myself into thinking that it contained jsonnet *code*.

File: rust/agama-lib/share/examples/profile.jsonnet [hunk 3]
```diff
@@ -1,4 +1,18 @@
...
+// For the schema, see
+// https://github.com/openSUSE/agama/blob/master/rust/agama-lib/share/profile.schema.json
+
+// The "hw.libsonnet" file contains hardware information of the storage devices
+// from the "lshw" tool. It is generated by running as root:
```
Comment (lslezak) [inline rust/agama-lib/share/examples/profile.jsonnet:7]: I think the most important information is that it is explicitly run by Agama itself, there is no magic for that.
https://github.com/openSUSE/agama/blob/b4bf25fdffd46735800a8396f1715fffe181e1db/rust/agama-lib/src/profile.rs#L128

Comment (imobachgs) [reply rust/agama-lib/share/examples/profile.jsonnet:7]: I merged too soon. I opened https://github.com/openSUSE/agama/pull/655.

**Pull request merged**